js/build.js

```diff
@@ -13,7 +13,24 @@ function init(){
       paginationClickable: true,
       nextButton: '.swiper-button-next-' + id,
       prevButton: '.swiper-button-prev-' + id,
-      grabCursor: true
+      grabCursor: true,
+      onSlideChangeEnd: function () {
+        /**
+         * get current page context if any
+         */
+        var existingPageContext = Fliplet.Env.get('pageContext') || {};
+        /**
+         * use a direct access data structure for faster lookup later
+         */
+        /**
+         * store the current slider context under the slider's id
+         */
+        var slidersContext = Object.assign(existingPageContext.sliders || {}, {
```

Comment:
`Object.assign()` does not appear to be supported in IE 11. [See link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility).

Please use lodash if it's not supported in IE 11.